frontend/components/selectSiteEngine.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const availableEngines = [
+  {
+    label: 'Jekyll',
+    value: 'jekyll',
+  },
+  {
+    label: 'Hugo',
```

Comment:
Maybe we could change the label to `Hugo (experimental)` or something like that?